pkg/timeline/adaptor/controller/timeline.ts

```diff
@@ -63,6 +63,56 @@ export class TimelineController {
     this.homeTimeline = args.homeTimeline;
   }
 
+  private async getNoteAdditionalData(notes: Note[]): Promise<
```

Comment:
```suggestion
  private async getNoteAdditionalData(notes: readonly Note[]): Promise<
```
`readonly` にできると思うので, できる限りそうしておきたいです.